vms/platformvm/state/staker_status.go

```diff
@@ -4,9 +4,10 @@
 package state
 
 const (
-	unmodified diffValidatorStatus = iota
+	unmodified diffStakerStatus = iota
```

Comment:
we can add/delete/update both validators and delegators. So renamed `diffValidatorStatus` to `diffStakerStatus`